test/verify/files/mock-insights

```diff
@@ -124,6 +125,51 @@ class handler(BaseHTTPRequestHandler):
             self.wfile.write(b'{ "reports": [ "foo", "bar" ] }\n')
             return
 
+        m = self.match("/r/insights/platform/ingress/v1/upload")
+        if m:
+            # The metadata of the system is in the multipart MIME data
+            # sent by the system for the upload; to pick it and use it
+            # we need to unpack the multipart MIME data.
+
+            # First, create prologue to the data, so it can be recognized
+            # as multipart MIME.
+            multipart_data = (
+                b"""MIME-Version: 1.0
+Content-Type: """
+                + self.headers.get("content-type").encode("utf-8")
+                + b"""
+
+"""
+            )
+            multipart_data += data
+
+            # Parse the multipart MIME data, and then look for a "metadata"
+            # file part which contains the system metadata as JSON.
+            message = email.message_from_bytes(multipart_data)
+            for part in message.walk():
+                if part.get_filename() == "metadata":
+                    s = json.loads(part.get_payload())
+                    s["id"] = "123-nice-id"
+                    print(s)
```

Comment:
debugging leftover

Reply:
there is the same in the other endpoint that "uploads" a system, so it's consistent :P i find it useful to check the log file for the result of the registration, especially in this case that the metadata is even more "entangled"